devices/common_patches/h7_dbgmcu_dualcore.yaml

```diff
@@ -0,0 +1,170 @@
+# Add dualcore parts of DBGMCU periperal. From RM0399 rev 2
+
+DBGMCU:
+  CR:
+    _add:
+      DBGSTBY_D2:
+        description: Allow debug in D2 Standby mode
+        bitOffset: 5
+        bitWidth: 1
+      DBGSTOP_D2:
+        description: Allow debug in D2 Stop mode
+        bitOffset: 4
+        bitWidth: 1
+      DBGSLEEP_D2:
+        description: Allow debug in D2 Sleep mode
+        bitOffset: 3
+        bitWidth: 1
+  APB4FZ1:
+    _add:
+      IWDG2:
+        description: Independent watchdog for D2 stop when Cortex-M7 in debug mode
+        bitOffset: 19
+        bitWidth: 1
+  APB1LFZ1:
+    _add:
+      DBG_WWDG2:
+        description: WWDG2 stop when Cortex-M7 in debug mode
+        bitOffset: 11
+        bitWidth: 1
+  _add:
+    APB3FZ2:
+      description: APB3 peripheral freeze register CPU2
+      addressOffset: 0x038
+      access: read-write
+      resetValue: 0x00000000
+      fields:
+        WWDG1:
+          description: WWDG1 stop when Cortex-M4 in debug mode
+          bitOffset: 6
+          bitWidth: 1
+    APB1LFZ2:
+      description: APB1L peripheral freeze register CPU2
+      addressOffset: 0x040
+      access: read-write
+      resetValue: 0x00000000
+      fields:
+        I2C3:
+          description: I2C3 SMBUS timeout stop when Cortex-M4 in debug mode
+          bitOffset: 23
+          bitWidth: 1
+        I2C2:
+          description: I2C2 SMBUS timeout stop when Cortex-M4 in debug mode
+          bitOffset: 22
+          bitWidth: 1
+        I2C1:
+          description: I2C1 SMBUS timeout stop when Cortex-M4 in debug mode
+          bitOffset: 21
+          bitWidth: 1
+        DBG_WWDG2:
```

Comment:
As with previous comment, should this be `WWDG2` without `DBG_`?

Reply:
Following the RM is the default, but useless prefixes are bad too. For minimal changes, I say strip the prefix